paper-tooltip.html

```diff
@@ -201,6 +202,12 @@
         _showing: {
           type: Boolean,
           value: false
+        },
+
+        /** @type {?EventTarget} */
+        _target: {
```

Comment:
This changes as a result of `forChanged`, right? Can we call its code in `forChanged` and the other 2 places, please? We don't really need another observer (which isn't free)